procmon/processmonitor.cpp

```diff
@@ -6618,32 +6540,29 @@ int ProcessMonitor::glusterUnassign(std::string dbrootID)
     std::string errmsg = "";
 
     log.writeLog(__LINE__, "glusterUnassign called: " + dbrootID, LOG_TYPE_DEBUG);
-
-    if ( rootUser)
-    {
-        command = "umount -f " + startup::StartUp::installDir() + "/data" + dbrootID + " > /tmp/glusterUnassign.txt 2>&1";
-    }
-    else
-    {
-        command = "sudo umount -f " + startup::StartUp::installDir() + "/data" + dbrootID + " > /tmp/glusterUnassign.txt 2>&1";
-    }
+    
+	string tmpLog = tmpLogDir + "/glusterUnassign.log";
+	
+    command = "umount -f " + startup::StartUp::installDir() + "/data" + dbrootID + " > " + tmpLog + " 2>&1";
 
     int ret = system(command.c_str());
 
     if ( WEXITSTATUS(ret) != 0 )
     {
 		log.writeLog(__LINE__, "glusterUnassign mount failure: dbroot: " + dbrootID + " error: " + oam.itoa(WEXITSTATUS(ret)), LOG_TYPE_ERROR);
 
-        ifstream in("/tmp/glusterUnassign.txt");
+        ifstream in(tmpLog);
         in.seekg(0, std::ios::end);
         int size = in.tellg();
 
         if ( size != 0 )
         {
-            if (!oam.checkLogStatus("/tmp/glusterUnassign.txt", "not mounted"))
+            if (!oam.checkLogStatus(tmpLog, "not mounted"))
             {
                 log.writeLog(__LINE__, "glusterUnassign failed.", LOG_TYPE_ERROR);
-                system("mv -f /tmp/glusterUnassign.txt /tmp/glusterUnassign_failed.txt");
+                             
+                string cmd = "mv -f " + tmpLog + " " + tmpLog + "failed";
```

Comment:
doesn't matter as much as the other stuff, but the before and after txt doesn't match.  old ver is a string literal, so easy to see it.  If tmpLog is 'log.txt', then the new version will construct 'mv -f log.txt log.txtfailed'